custom/icds_reports/data_pull/sql_queries/create_tmp_pse_table.sql

```diff
@@ -1,5 +1,5 @@
 DROP TABLE IF EXISTS temp_pse_data_pull;
-CREATE TEMPORARY TABLE temp_pse_data_pull AS
+CREATE TABLE temp_pse_data_pull AS
```

Comment:
hmm, this would leave the table in storage forever and consume space.
Would be nice to add a cleanup step which cleans the table after the data has been collected.